src/zst_decompressor.ts

```diff
@@ -108,9 +114,10 @@ const decompressAndAnalyze = async (file, start = 0) => {
 
       // https://www.npmjs.com/package/node-zstandard#decompressionstreamfromfile-inputfile-callback
       zstd.decompressionStreamFromFile(
-        `${__dirname}/../../data/${file}`,
+        `${compressedFilePath}/${file}`,
```

Comment:
fyi these aren't bugs. They are differences between your workflow using ts-node and mine which transpiles to JS first.

Reply:
understood. can we write code to work with both?